From d149b07b41e45b5a9c3700b72835626f89714171 Mon Sep 17 00:00:00 2001
From: Han Wang <zjuwanghan@outlook.com>
Date: Sun, 15 Aug 2021 20:34:39 +0800
Subject: [PATCH -next] maple: remove useless INIT_LIST_HEAD()

list maple_waitq and maple_sentq have been inited staticly
through LIST_HEAD. So there's no need to call INIT_LIST_HEAD
again, remove it.

Signed-off-by: Han Wang <zjuwanghan@outlook.com>
---
 drivers/sh/maple/maple.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/sh/maple/maple.c b/drivers/sh/maple/maple.c
index e24e220e5..44f00d7cc 100644
--- a/drivers/sh/maple/maple.c
+++ b/drivers/sh/maple/maple.c
@@ -839,9 +839,6 @@ static int __init maple_bus_init(void)
 		goto cleanup_bothirqs;
 	}
 
-	INIT_LIST_HEAD(&maple_waitq);
-	INIT_LIST_HEAD(&maple_sentq);
-
 	/* setup maple ports */
 	for (i = 0; i < MAPLE_PORTS; i++) {
 		checked[i] = false;
-- 
2.25.1

